src/Input/Input.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import * as cx from 'classnames';
+import { props, t, ReactChildren } from '../utils';
+import omit = require('lodash/omit');
+import InputChildren from 'react-input-children';
+import { ObjectOverwrite } from 'typelevel-ts';
+
+export type InputRequiredProps = ObjectOverwrite<React.HTMLProps<HTMLInputElement>, {
+  /** value */
+  value: string,
+  /** onChange */
+  onChange: (value: string) => void,
```

Comment:
is there a case where we want a static input with only `defaultValue`? (maybe in addition with `disabled` `true`)

if yes, maybe we should make `value` and `onChange` optional? what do you think?

Reply:
static input -> `value='string' & onChange=() => {}`
static input with "placeholder" style... 🤷‍♂️ 

I'd leave it as is for the moment